design-notes/on-teaching-operators.md

```diff
@@ -73,7 +73,7 @@ These operators are found in F# code but can normally be left off a teaching pat
 with a simple google search:
 
     %                                Modulus
-    &&& ||| ^^^                      Bitwise (&& in C#)
+    &&& ||| ^^^                      Logical (&& in C#)
```

Comment:
I reckon you're half right ;) He means bitwise (those operators are less frequently used), but you are correct that he's mentioned the wrong C# operators 
```suggestion
    &&& ||| ^^^                      Bitwise (&, |, ^ in C#)
```

Reply:
Thank you for the correction. Recently started F# learning. :)